documentation/src/main/markdown/scripts/overview.md

```diff
@@ -27,13 +27,13 @@ Refer to [Running the Synopsys Detect script](../runningdetect/basics/runningscr
 | DETECT_JAR_PATH | Change the [solution_name] .jar download dir | The path to the .jar file download directory | If DETECT_JAR_PATH is provided, the script will use this location when downloading and running detect. The location of the jar will be DETECT_JAR_PATH/[source_project_name]-{version}.jar. The Bash script will default to '{user home directory}/synopsys-detect/download' if no option is specified. |
 | TMP (PowerShell only) | Provides the user's temporary directory | The path to a directory for temporary files | If DETECT_JAR_PATH is not provided, the script will use the environment 'TMP' variable as the folder for the [solution_name] .jar path. |
 | HOME (PowerShell only) | Provides the user's home directory | The path to the user's home directory | If DETECT_JAR_PATH is not provided and no 'TMP' variable can be found, the '$HOME/tmp' folder will be used for the [solution_name] jar path. |
-| DETECT_JAVA_PATH | The path to the java executable to use to execute [solution_name] | The path to the java executable file | |
+| DETECT_JAVA_PATH | The path to the Java instance used to execute Docker Inspector | Path to the Java executable file. | To set the Java instance used by [solution_name], see JAVA_HOME. |
```

Comment:
Nit: It is also possible for the customer to invoke Detect with a specific Java by using the specific Java executable to invoke Detect.